app/models/journeys/page_sequence.rb

```diff
@@ -5,7 +5,7 @@ module Journeys
   class PageSequence
     attr_reader :current_slug
 
-    DEAD_END_SLUGS = %w[complete existing-session eligible-later future-eligibility ineligible]
+    DEAD_END_SLUGS = %w[complete existing-session eligible-later future-eligibility ineligible check-your-email]
```

Comment:
Out of scope for this PR but I don't think `DEAD_END_SLUGS` should be a constant in the page sequence, IMO it should be journey specific.